rules/falco_rules.yaml

```diff
@@ -1936,17 +1941,84 @@
     WARNING
   tags: [process, mitre_credential_access]
 
+- list: log_directories
+  items: [/var/log, /dev/log]
+
+- list: log_files
+  items: [syslog, auth.log, secure, kern.log, cron, user.log, dpkg.log, last.log, yum.log, access_log, mysql.log, mysqld.log]
+
+- macro: access_log_files
+  condition: (fd.directory pmatch (log_directories) or fd.filename in (log_files))
+
+- rule: Clear Log Activities
+  desc: Detect clearing of critical log files
+  condition: >
+    ((open_write and access_log_files and evt.arg.flags contains "O_TRUNC") or
+     (remove and access_log_files))
```

Comment:
As we just discussed, unlinks don't have an active fd--the path is just an argument to the syscall. We should limit rules like this to only writes until we can efficiently break up path arguments into directory and/or filename components.